frontend/src/components/App/FishbowlList/index.tsx

```diff
@@ -230,7 +228,7 @@ const FishbowlList: React.FC<Props> = ({ selectedFishbowlParam }) => {
                       <motion.div
                         className="form-wrapper"
                         variants={
-                          windowWidth <= BREAKPOINTS.desktop
+                          windowWidth !== undefined && windowWidth <= BREAKPOINTS.desktop
```

Comment:
Is this logic desired? if windowWidth is undefined we will pick basicRevealWithDelayVariant. 

Same apply for the others.